section11/numberformat_diff.html

```diff
@@ -0,0 +1,1047 @@
+            <td>`"currencyDisplay"`</td>
+          </tr>
+          <tr>
+            <td><ins class="block">[[CurrencySign]]</ins></td>
+            <td><ins class="block">`"currencySign"`</ins></td>
+          </tr>
+          <tr>
+            <td>[[MinimumIntegerDigits]]</td>
+            <td>`"minimumIntegerDigits"`</td>
+          </tr>
+          <tr>
+            <td>[[MinimumFractionDigits]]</td>
+            <td>`"minimumFractionDigits"`</td>
+          </tr>
+          <tr>
+            <td>[[MaximumFractionDigits]]</td>
+            <td>`"maximumFractionDigits"`</td>
+          </tr>
+          <tr>
+            <td>[[MinimumSignificantDigits]]</td>
+            <td>`"minimumSignificantDigits"`</td>
+          </tr>
+          <tr>
+            <td>[[MaximumSignificantDigits]]</td>
+            <td>`"maximumSignificantDigits"`</td>
+          </tr>
+          <tr>
+            <td><ins class="block">[[Notation]]</ins></td>
+            <td><ins class="block">`"notation"`</ins></td>
+          </tr>
+          <tr>
+            <td><ins class="block">[[CompactDisplay]]</ins></td>
+            <td><ins class="block">`"compactDisplay"`</ins></td>
+          </tr>
+          <tr>
+            <td>[[UseGrouping]]</td>
+            <td>`"useGrouping"`</td>
+          </tr>
+          <tr>
+            <td><ins class="block">[[SignDisplay]]</ins></td>
+            <td><ins class="block">`"signDisplay"`</ins></td>
+          </tr>
+        </table>
+      </emu-table>
+    </emu-clause>
+  </emu-clause>
+
+  <emu-clause id="sec-properties-of-intl-numberformat-instances">
+    <h1>Properties of Intl.NumberFormat Instances</h1>
+
+    <p>
+      Intl.NumberFormat instances inherit properties from %NumberFormatPrototype%.
+    </p>
+
+    <p>
+      Intl.NumberFormat instances have an [[InitializedNumberFormat]] internal slot.
+    </p>
+
+    <p>
+      Intl.NumberFormat instances also have several internal slots that are computed by the constructor:
+    </p>
+
+    <ul>
+      <li>[[Locale]] is a String value with the language tag of the locale whose localization is used for formatting.</li>
+      <li><ins class="block">[[DataLocale]] is a String value with the language tag of the nearest locale for which the implementation has data to perform the formatting operation. It will be a parent locale of [[Locale]].</ins></li>
+      <li>[[NumberingSystem]] is a String value with the "type" given in Unicode Technical Standard 35 for the numbering system used for formatting.</li>
+      <li>[[Style]] is one of the String values `"decimal"`, `"currency"`, <del>or</del> `"percent"`<ins>, or `"unit"`</ins><del>, identifying the number format style used</del>. <ins>This value corresponds closely with the [[UnitType]] value.</ins></li>
+      <li><ins class="block">[[UnitType]] is one of the String values `"none"`, `"currency"`, or `"measureUnit"`. The value `"measureUnit"` is used for both [[Style]] `"percent"` and `"unit"`.</ins></li>
```

Comment:
I *think* I agree with @littledan that it might be clearer to just have `[[Style]]` and introduce an abstract operation to get the "UnitType".  If that's possible!

Reply:
Ok.  Two votes, and there isn't a super-strong reason to keep `[[UnitType]]`.  Fixed.